### PR TITLE
Revamp for Troly WP Plugin UI

### PR DESCRIPTION
**Pull request by rcdtroly**
### Why:
- There's a need to improve how the settings in the Troly plugin are displayed. Users may not scroll down to look at other options. Consider using tabbed page.

### What:
- applied Initial layout base to this mockup: https://docs.google.com/drawings/d/1D-gqV8zKNi9ZTgwmdNKmFkeiy_eP1n2UxM94rtfdPwU/edit?usp=sharing
- applied redesign base on miro board: https://realtimeboard.com/app/board/o9J_kxj4B5A=/

### Refs:
- https://app.asana.com/0/939825968139797/895754844747192
- https://app.asana.com/0/939825968139797/951209539261060

### Result(s):
- New tab version of the Troly settings.
Alt text

- New tab version for Operations
Alt text

### CODE REVIEW

**Comment by rcdtroly** [comment]
added **Enable "member login" link to show member pricing**
Alt text

**Comment by rcdtroly** [comment]
Connection to Troly
Alt text

Operations Page
Alt text

File: includes/admin/assets/js/wizard-steps.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+jQuery(function ()
```
Comment (JamesEmpireOne) [inline includes/admin/assets/js/wizard-steps.js]: If the wizard is not being rolled out, can we remove this file?

File: includes/admin/controllers/class-wp99234-admin-menu.php [hunk 2]
```diff
@@ -87,6 +88,14 @@ public function operations_page_init() {
 	public function operations_page_html() {
 		WP99234_Admin_operations::output();
 	}
+
```
Comment (JamesEmpireOne) [inline includes/admin/controllers/class-wp99234-admin-menu.php]: If the wizard functionality isn't going out in this release, can we put these changes into another branch?

File: includes/admin/controllers/class-wp99234-admin-menu.php [hunk 3]
```diff
@@ -27,6 +27,7 @@ public function __construct() {
 		add_action( 'admin_menu', array( $this, 'admin_menu' ), 9 );
 		add_action( 'admin_menu', array( $this, 'settings_menu' ), 50 );
 		add_action( 'admin_menu', array( $this, 'operations_menu' ), 20 );
+        // add_action( 'admin_menu', array( $this, 'wizard_menu' ), 60 );
```
Comment (JamesEmpireOne) [inline includes/admin/controllers/class-wp99234-admin-menu.php]: If the wizard functionality isn't going out in this release, can we put these changes into another branch?

File: includes/admin/controllers/class-wp99234-admin.php [hunk 4]
```diff
@@ -100,6 +100,7 @@ public function buffer() {
     public function includes() {
         include_once( 'class-wp99234-admin-page.php' );
         include_once( 'class-wp99234-admin-settings.php' );
+        include_once( 'class-wp99234-admin-wizard.php' );
```
Comment (JamesEmpireOne) [inline includes/admin/controllers/class-wp99234-admin.php]: If the wizard functionality isn't going out in this release, can we put these changes into another branch?

File: includes/admin/controllers/class-wp99234-admin-wizard.php [hunk 5]
```diff
@@ -0,0 +1,109 @@
+<?php
```
Comment (JamesEmpireOne) [inline includes/admin/controllers/class-wp99234-admin-wizard.php]: If the wizard functionality isn't going out in this release, can we put these changes into another branch?

File: includes/admin/views/html-admin-wizard.php [hunk 6]
```diff
@@ -0,0 +1,41 @@
+<?php
```
Comment (JamesEmpireOne) [inline includes/admin/views/html-admin-wizard.php]: If the wizard functionality isn't going out in this release, can we put these changes into another branch?

File: wp99234.php [hunk 7]
```diff
@@ -1333,3 +1333,38 @@ function xtheme_troly_credits( $string ) {
     // Add Troly footer
     return $string . ' Powered by Troly <span style="color:red;">&#10084;</span>';
 }
+
```
Comment (JamesEmpireOne) [inline wp99234.php]: If the wizard functionality isn't going out in this release, can we put these changes into another branch?

**Comment by rcdtroly** [comment]
Enable or Disable disclaimer related fields
Alt text

**Pull request merged**